docs-source/content/release-notes.md

```diff
@@ -8,16 +8,17 @@ draft: false
 
 | Date | Version | Introduces backward incompatibilities | Change |
 | --- | --- | --- | --- |
-| February 26, 2020 | v2.5.0 | no | Support for Helm 3.x and OpenShift 4.3.  Operator can be installed in a namespace-dedicated mode where operator requires no cluster-level Kubernetes privileges. This version is not supported on Kubernetes 1.16+, check the prerequisites.
-| November 15, 2019 | v2.4.0 | no | Includes fixes for a variety of issues related to FMW infrastructure domains and pod variable substitution.  Operator now uses WebLogic Deploy Tooling 1.6.0 and the latest version of the Kubernetes Java Client. 
+| May 8, 2020 | v3.0.0-rc1 | yes; not compatible with earlier releases. | Adds Model in Image feature. Support for Kubernetes 1.16. Deprecation of support for Kubernetes 1.13 and earlier. Deprecation of support for Helm 2.x. |
```

Comment:
'not compatible with earlier releases' sounds a little harsh to me, and maybe even a bit misleading if read out of context - it's not 'rolling' compatible for sure but I assume it's upgrade compatible?

Reply:
That's right.  The important bit is that we can't support running an operator of version 2.5.0 and one of 3.0.0-rc1 in the same cluster.

Reply:
OK. I see it's now `yes; cannot coexist in the same cluster with earlier releases`.  

Can it be a bit more detailed here?  'an earlier release's domains can be deployed to this release, but an earlier release's operator cannot co-exist in the same cluster'

Reply:
how's this new change @tbarnes-us

Reply:
LGTM.